File: Client/src/client/systems/blips.js
/// <reference types="@altv/types-client" />
/// <reference types="@altv/types-natives" />

import alt from 'alt-client';
import native from 'natives';

alt.onServer('Client:SendFactionList', (PosX, PosY, PosZ, BlipId, BlipColorId, Name) => {
    alt.log("Test");
    createBlip(PosX, PosY, PosZ, BlipId, BlipColorId, Name);
});

function createBlip(posX, posY, posZ, blipId, blipColorId, name) {
    const blip = new alt.PointBlip(posX, posY, posZ);
    blip.sprite = blipId;
    blip.color = blipColorId;
    blip.name = name;
    blip.display = 3;
}

File: Los Angeles Life Client/Los Angeles Life Client/System/Blips.cs
using AltV.Net.Client;
using AltV.Net.Data;

namespace Los_Angeles_Life_Client.System;

public abstract class Blips : Client
{
    public static void Load()
    {
        Alt.OnServer("Client:SendBlipList", (float positionX, float positionY, float positionZ, ushort blipId, byte blipClorId,
            string name) =>
        {
            CreateBlip(positionX, positionY, positionZ, blipId, blipClorId, name);
        });
    }

    private static void CreateBlip(float positionX, float positionY, float positionZ, ushort blipId, byte blipClorId,
        string name)
    {
        var blip = Alt.CreatePointBlip(new Position(positionX, positionY, positionZ));
        blip.Sprite = blipId;
        blip.Color = blipClorId;
        blip.Name = name;
        blip.Display = 3;
    }
}